pkg/traces/remotewriteexporter/factory.go

```diff
@@ -13,20 +14,15 @@ const (
 	TypeStr = "remote_write"
 )
 
-type label struct {
-	Name  string `mapstructure:"name"`
-	Value string `mapstructure:"name"`
-}
-
 var _ config.Exporter = (*Config)(nil)
 
 // Config holds the configuration for the Prometheus remote write processor.
 type Config struct {
 	config.ExporterSettings `mapstructure:",squash"`
 
-	ConstLabels  []label `mapstructure:"const_labels"`
-	Namespace    string  `mapstructure:"namespace"`
-	PromInstance string  `mapstructure:"metrics_instance"`
+	ConstLabels  *prometheus.Labels `mapstructure:"const_labels"`
```

Comment:
Why use a pointer to `prometheus.Labels`? It makes handling later a bit more complicated having to dereference it.

Reply:
I just made it the same signature as in https://github.com/grafana/agent/blob/main/pkg/traces/config.go#L255
But that is probably not necessary, I'll switch to a non-pointer!